jsBasics/task4_1.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+let radius = 13;
+let results = Math.PI * (radius**2);
+console.log(`Площа кола за формулою π * radius^2 = ${results}`);
```

Comment:
В завданні сказано, округлити до двох знаків після коми кожне значення, а не тільки останнє
```suggestion
console.log(`Площа кола за формулою π * radius^2 = ${results.toFixed(2)}`);
```